internal/domain/power-status.go

```diff
@@ -479,6 +485,10 @@ func getHWStatesFromHW() error {
 					taskList[taskIX].Request.Header.Add(hashXName, k)
 					taskList[taskIX].Request.Header.Add(hashCType, string(ctype))
 					taskList[taskIX].Request.Header.Add(hashFQDN, v.HSMData.RfFQDN)
+
+					taskList[taskIX].Timeout = time.Second * time.Duration(httpTimeout)
+					taskList[taskIX].RetryPolicy.Retries = httpRetries
+
```

Comment:
These lines are a little redundant with line 440.  You could change line 440:

`    sourceTL := trsapi.HttpTask{Timeout: time.Duration(httpTimeout) * time.Second}`

To this:

`    sourceTL := trsapi.HttpTask{Timeout: time.Duration(bmcTimeout) * time.Second, `
`                                RetryPolicy.Retries: httpRetries}`

Because sourceTL is copied into each task as part of line 456:

    `taskList := (*tloc).CreateTaskList(&sourceTL, len(hwStateMap))`